packages/trpc/server/admin-router/schema.ts

```diff
@@ -48,6 +48,20 @@ export const ZAdminDeleteUserMutationSchema = z.object({
 
 export type TAdminDeleteUserMutationSchema = z.infer<typeof ZAdminDeleteUserMutationSchema>;
 
+export const ZAdminEnableUserMutationSchema = z.object({
+  id: z.number().min(1),
+  email: z.string().email(),
+});
+
+export type TAdminEnableUserMutationSchema = z.infer<typeof ZAdminEnableUserMutationSchema>;
+
+export const ZAdminDisableUserMutationSchema = z.object({
+  id: z.number().min(1),
+  email: z.string().email(),
+});
```

Comment:
Why are we passing email in? Can't we just use the ID?

Reply:
Same functionality as deleting the user

Reply:
It doesn't make sense either way, remove the email and just use the id of the user to make it cleaner.

Reply:
Already uses the ID to retrieve the user. 

The email is used to check if the retrieved user's email matches the provided email.

Will remove it though.